client/src/components/Login.js

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from "react";
+import axios from "axios";
+import Typography from "@material-ui/core/Typography";
+import Container from "@material-ui/core/Container";
+import Paper from "@material-ui/core/Paper";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+
+export default function Login({ setIsLogin }) {
+  const [user, setUser] = useState({ name: "", email: "", password: "" });
+  const [error, setError] = useState("");
+
+  const onChangeInput = (e) => {
+    const { name, value } = e.target;
+    setUser({ ...user, [name]: value });
+    setError("");
+  };
+
+  const registerSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const res = await axios.post("/users/register", {
```

Comment:
extract this to an api folder. Maybe something like userApi.js inside a folder called "api"